File: TeachingAppAPI/Client-App/src/app/create-course/course-content/lesson.ts
export class Lesson{

    lessonId: number;
    lessonName: string;
    lessonOrder: number;
    lessonArticle: string;
    lessonVideoUrl: string;
    videoFileName: string;
    s3VideoFileName: string;

    constructor(lessonId, lessonName, lessonOrder, lessonArticle, lessonVideoUrl, videoFileName, s3VideoFileName){
        this.lessonId = lessonId;
        this.lessonName = lessonName;
        this.lessonOrder = lessonOrder;
        this.lessonArticle = lessonArticle;
        this.lessonVideoUrl = lessonVideoUrl;
        this.videoFileName = videoFileName;
        this.s3VideoFileName = s3VideoFileName;
    }
}

File: TeachingAppAPI/Entities/Lesson.cs
using System;
using System.Collections.Generic;

namespace TeachingAppAPI.Entities
{
    public partial class Lesson
    {
        public int LessonId { get; set; }
        public int? TopicId { get; set; }
        public string LessonName { get; set; }
        public string LessonArticle { get; set; }
        public string LessonVideoUrl { get; set; }
        public int? LessonOrder { get; set; }
        public string VideoFileName { get; set; }
        public string S3VideoFileName { get; set; }

        public Topic Topic { get; set; }
    }
}
